core/src/test/scala/com/salesforce/op/OpWorkflowTest.scala

```diff
@@ -241,14 +241,16 @@ class OpWorkflowTest extends FlatSpec with PassengerSparkFixtureTest {
     val fv = Seq(age, gender, height, weight, description, boarded, stringMap, numericMap, booleanMap).transmogrify()
     val survivedNum = survived.occurs()
     val pred = BinaryClassificationModelSelector().setInput(survivedNum, fv).getOutput()
+
     val wf = new OpWorkflow()
       .setResultFeatures(pred)
-      .withRawFeatureFilter(Option(dataReader), Option(simpleReader),
-        maxFillRatioDiff = 1.0) // only height and the female key of maps should meet this criteria
+      .withRawFeatureFilter(Option(dataReader), None, maxFillRatioDiff = 1.0)
     val data = wf.computeDataUpTo(weight)
 
+    // Since there are < 500 rows in the scoring set, only the training set checks are applied here, and the only
+    // removal reasons should be null indicator - label correlations
     data.schema.fields.map(_.name).toSet shouldEqual
-      Set("key", "height", "survived", "stringMap", "numericMap", "booleanMap")
+      Set("booleanMap", "description", "height", "stringMap", "age", "key", "survived", "numericMap")
```

Comment:
once things are parameterized lets set them so that the tests remain the same

Reply:
fixed

Reply:
I was just about to suggest the same ;)